### ISSUE TITLE
Блок аналитики в конструкторе

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2600]
Originally reported on Google Code with ID 2600

```
Есть проблема, когда переносятся проводки на базу, где аналитика по счетам стоит не
такая как на исходной базе. Тогда при редактировании данного блока заполнение аналитик
пропадает, что иногда ведёт к плохим последствиям. Я добавил обработку данной ситуации,
посмотрите если интересно
```

Reported by `Alexander.GoldenSoft` on 2011-10-16 10:23:39

**Comment by gsbelarus** [comment]
```

  Если, я правильно понимаю, то замечена следующая ситуация. 
  Выполнили перенос ТХО через настройку из базы данных 1 в базу данных 2. Простая проводка:
Дт - 60.01 - Кт 90.01. При этом, на базе 1 по счету 60.01 проставлена аналитика: "Клиент",
"Документ", "Тип НДС", "Договор". На базе 2 по счету 60.01 проставлена аналитика: "Клиент",
"Документ", "Тип НДС". Здесь нет аналитики "Договор". На базе данных 2 пробуем сформровать
запись по соответствующему документу, в результате программа выдает сообщение об ошибке
в проводках. Редактируем проводки и обнаруживаем: в блоках по счету 60.01 нет аналитики
"Договор", а в правой части (скрипты) есть запись по данной аналитике. Для решения
вопроса необходимо перебить любую из аналитик и тогда состояние скриптов будет соответствовать
состоянию блоков.

```

Reported by `NikolayUkleyko` on 2013-11-12 10:05:43

**Comment by gsbelarus** [comment]
Reported by `NikolayUkleyko` on 2013-12-19 08:41:46
- **Status changed**: `Duplicate`
- **Merged into**: #3278
